docs/user/BYOnetwork.md

```diff
@@ -0,0 +1,23 @@
+## Supported scenarios and requirements for using existing AWS network resources
+
+The table below shows the scenarios supported with using existing/bring your own(BYO) network resources:
+
+| <div style="width:70px">Scenario</div> | <div style="width:70px">BYO resource</div> | <div style="width:500px">Requirements</div> | <div style="width:100px">Resources that will be created</dvi>|
+| :--- | :--- | :--- | :--- |
+| 1 | VPC | <ul><li>VPC block size must be IPv4 CIDR block with '/16' netmask (supports 65,536 IP addresses)</li><li>`DNS hostnames` and `DNS resolution` are enabled</li><li>`var.vpc_id` must be provided</li><li>`var.subnets` CIDR blocks must match with VPC CIDR IPv4 CIDR block</li></ul> | Subnets (NAT_id, routes and Route Table association) and Security Group|
```

Comment:
Why are is this bullet under Requirements for scenarios 1 vpc? "var.subnets CIDR blocks must match with VPC CIDR IPv4 CIDR block". If viya4-iac-aws will create the subnets for me, why would the requirements matter? Should this bullet be in the requirements for scenario 2?

Reply:
This is 'var.subnets' (that's different from BYO 'var.subnet_ids') - https://github.com/sassoftware/viya4-iac-aws/blob/byo-custom-module/variables.tf#L269 that must match to VPC's CIDR block in case of BYO VPC id.